mechanical/mechanical_integration/README.md

```diff
@@ -1,32 +1,154 @@
-# Mechanical Integration
-This is the final mechanical integration of all of the other subassemblies to create the final rover.
 
-<img src="/images/mechanical_integration/final_title.png" width="80%">
 
-## Features
-  * All subassemblies can be combined together with the addition of very few additional components.
+#  1 Mechanical/Structural Assembly
+At this point we have every major sub assembly created and they just need to be attached together to create the rover! In the steps in this document, we will attach all the corner motors and wheels, the rocker-bogie, the differential pivot, and the head.
 
-## Information about module
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/differential_pivot.png">
+	</td>
+	<td><img src="../../images/mechanical_integration/head_final.png"></td>
+	<td><img src="../../images/mechanical_integration/wheels_title.png"></td>
+</tr>
+</table>
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/rocker_bogie_title.png"></td>
+	<td><img src="../../images/mechanical_integration/finished_body.png"></td>
+</tr>
+</table>
 
-| Author/Creator            | JPL       |
-| --------------            | -------            |
-| Cost without 3D printer   | 56.58              |
-| Cost with 3D printer      | 56.58              |
-| Date Added                | 4/11               |
-| Group/Organization/School | JPL                |
-| Time to build             | 2 hrs              |
+**Figure 1: All subassemblies completed**
 
-## Skills
+## 1.1 Mounting Wheels
+ **Table 1: Parts/Tools Necessary**
+ 
+| Item | Ref | Qty | Image|
+|------|-----|-----|------|
+|1.5” Channel|S1| 2|<img src="../../images/components/structural/S1.png" width="30%">
+|Channel Connector Plate|S6|14|<img src="../../images/components/structural/S6.png" width="30%">
+|0.5” x 15” Aluminum Tube (Modified)|S16A|1|<img src="../../images/components/structural/S16A.png" width="30%">
+|0.5” x 2” Aluminum Tube|S19|1|<img src="../../images/components/structural/S19.png" width="30%">
+|0.5” Collar Clamp|S22|7|<img src="../../images/components/structural/S22.png" width="30%">
+|0.5” Nylon Washer|W3|3| <img src="../../images/components/washers/W3.png" width="30%">
+|#6-32x1/4” Button Head Screw|B1|52| <img src="../../images/components/screws/B1.png" width="30%">
+|Wrench Set|D1||<img src="../../images/components/tools/D1.png" width="30%">
+|Allen Key Set|D2||<img src="../../images/components/tools/D2.png" width="30%">
 
-## Tools
 
-### Mandatory 
-  * Allen Key set
-  * Imperial Wrench Set
+  
 
-### Optional
+1. Mounting the Middle Wheel: Mount one base wheel (not the corner wheels) to the lowest arm of each of the two rocker bogie subassemblies using a 1.5 inch channel **S1** and channel connectors S6. See Figures 2 and 3. Note that you should mirror the wheels so that they are both on the ”outside” of the rocker bogey subassemblies.
```

Comment:
also would be good to follow the N.N.N numbering scheme from https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md for these sub-subsections

helps to keep the reader oriented for where they are in the file

Reply:
done